packages/mdc-switch/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@material/switch",
+  "description": "The Material Components for the web switch component",
+  "version": "0.3.0",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "dependencies": {
+    "@material/animation": "^0.1.3",
+    "@material/elevation": "^0.1.2",
+    "@material/theme": "^0.1.1"
+  }
```

Comment:
can you add

```
"publishConfig": {
  "access": "public"
}
```

here? 

This way lerna will be able to publish the scoped packages.